firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -32,17 +35,23 @@ enum Quarters{
 // Makes a full circle of the given size (0=largest, NUM_CIRCLES-1=smallest) 
 // starting from the given quadrant (0=top, 1=left, 2=bottom, 3=right).
 // calculatePulsesPerSlope() must be called before makeCircle()
-void makeCircle(int circle, int firstQuarter)
+void makeCircle(int circle)
 {
+    // Loop counters
+    int slopeIndex, quarter, i;
+
     // firstPass is used to make the loop run for the first quadrant
     // despite the fact that quarter will be equal to first quarter.
     bool firstPass = true;
-    int slopeIndex;
+
+    // Circle 0 starts at the top, circle 1 at the left, circle 2 at the bottom, continuing in a counterclockwise fashion
+    // The % 4 is used in case there are more than four circles. Circle 4 should start at the top, 5 at the left, etc
+    int firstQuarter = circle % 4;
 
     setScale(xMotor, EIGHTH_STEPS);
     digitalWrite(xMotor[STEP_PIN], LOW);
 
-    for (int quarter = firstQuarter; quarter != firstQuarter || firstPass; quarter = (quarter + 1) % 4)
+    for (slopeIndex = firstQuarter; quarter != firstQuarter || firstPass; quarter = (quarter + 1) % 4)
```

Comment:
Just making sure, but should this be quarter?

Reply:
Thanks!